test/input-test.jl

```diff
@@ -8,7 +8,8 @@ end
 
 # Assert frequency constructor matches the rate constructor
 @testset "ConstantRate Constructors" begin

```

Comment:
We can group this with the previous test set into a single "ConstantRate" set that does both tests one after the other.